.github/workflows/reusable-workflow-notification.yml

```diff
@@ -0,0 +1,37 @@
+# this is useful because notifications for scheduled workflows are only sent to the user who
+# initially created the given workflow
+name: Reusable - Workflow notification
+
+on:
+  workflow_call:
+    inputs:
+      success:
+        type: boolean
+        required: true
+
+jobs:
+  workflow-notification:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Open issue or add comment if issue already open
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          number=$(gh issue list --search "Workflow failed: $GITHUB_WORKFLOW" --limit 1 --json number -q .[].number)
+
+          echo $number
+          echo ${{ inputs.success }}
```

Comment:
Nit: do we still need these debug prints?

Reply:
Copied this from opentelemetry-java-instrumentation. I'm fine removing them if @trask thinks the script logic is reliable. I did test this in my personal fork and it seemed to work ok, but didn't test edge cases.

Reply:
Ya I think we're good to remove